### DISCUSSION TITLE
Commits to my GitHub Page not showing up on my profile

### DISCUSSION THREAD

**Discussion by sararsaurus** [community/community#50841] [New to GitHub] [answered]
Hi! 

I'm a newbie and I work on my GitHub Page most days, but I just realized that those contributions don't show up on my profile. I followed the documentation and as far as I can tell, I'm making commits in a standalone repo, my commits are in the main branch, and my email address is correctly associated with my GitHub account. 

Any help would be much appreciated. Many thanks and my apologies in advance for whatever I've messed up!

**Comment by airtower-luna** [comment, marked as answer]
The commits aren't recognized as yours. GitHub uses the author email recorded in the commit to assign them to users. Looking at the author email it looks like there are quotes around your email address that definitely don't belong there. I can't say if the address is correct otherwise, double checking would make sense.

You can [set the correct email address](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity) for *future* commits like so (substitute your own address):

```
git config --global user.email you@example.com
```

You can [rewrite the repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to fix the address in old commits. Rewriting history can cause trouble if you've already shared it, but for a website you're working on on you own it should be fine. Double check before force-pushing, though, or you might lose data.

**Comment by sararsaurus** [comment]
Those quotes!! Thank you so much - problem solved!

**Comment by G271an** [comment]
dependabot/maven/org.jenkins-ci.plugins-plugin-4.56

**Comment by G271an** [comment]
#25245

**Comment by mdsaiful442** [comment]
i am a student
